Client/Services/HomeService.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+namespace Client.Services
+{
+    public  class HomeService
```

Comment:
Obviously no actual implementation here yet, but just fyi you will need to inject price API through constructor